ruby-gem/lib/calabash-android/drag_helpers.rb

```diff
@@ -0,0 +1,109 @@
+require 'calabash-android/env'
+
+module Calabash
+  module Android
+    module DragHelpers
+
+      MAX_RETRIES = 7
+      @@monkey_port = nil
+
+      def start_monkey
+        kill_existing_monkey_processes
+        retries = 0
+        loop do
+          port = rand((1025..65535))
+          output = ''
+          fork do
+            output = `#{Env.adb_path} shell monkey --port #{port}`
```

Comment:
@TobiasRoikjer I'm a bit stuck with this one - the call to `adb shell monkey` doesn't return unless there's an error. So I'd want to background the call, except then we also need a way to check if there was output. I'm not sure how to do this in ruby - I'm sure I could figure it out, but if you have a preferred approach it would save us both some time.